src/main/scala/akka/minion/GithubService.scala

```diff
@@ -195,11 +198,16 @@ class GithubService(settings: Settings, listeners: Seq[ActorRef]) extends Actor
 
   private val timer = context.system.scheduler.schedule(1.second, settings.pollInterval, self, Refresh)
 
+  private val responseCache = Scaffeine()
+    .maximumSize(500)
+    .build[Uri, (EntityTag, Any)]()
+
   // Throttled global connection pool
   val (queue: SourceQueueWithComplete[(HttpRequest, Promise[HttpResponse])], clientFuture: Future[Done]) =
     Source.queue[(HttpRequest, Promise[HttpResponse])](256, OverflowStrategy.backpressure)
       .throttle(settings.apiCallPerHour, 1.hour, 100, ThrottleMode.shaping)
       .via(Http(context.system).cachedHostConnectionPoolHttps("api.github.com"))
+      //.via(responseCache)
```

Comment:
Hm. Why is this commented?

Reply:
What are the things that this would cache?

Reply:
(maybe we don't want them cached across the board, is what I'm thinking)

Reply:
Or maybe I just have no clue how the minion works...

Reply:
AFAICR we want to smartly cache responses together with the github's support for `If-None-Match` and Etag support, since a response from github that indicates that nothing has changed does not count towards the quota.

Reply:
Ah. Thanks for explaining. so maybe the old impl didn't work and was therefore commented out? I mean, this PR fixes the caching, right?

Reply:
I see now. `responseCache` is also added in this PR. But it is used in `throttledJson`. Line 210 should just be removed.